src/style.css

```diff
@@ -1,3 +1,88 @@
+/* Reseta margens do navegador */
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  background-color: #fff;
+  color: #000;
+}
+
+/* Estilos para o cabeçalho */
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 40px;
+}
+
+/* Logo */
+.logo img {
+  width: auto;
+  height: 40px;
+}
+
+/* Estilos para a navegação */
+nav {
+  display: flex;
+}
+
+nav ul {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+/* Estilos para os links */
+nav a {
+  text-decoration: none;
+  color: #000;
+  margin: 0 20px;
+  position: relative;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: bold;
+}
+
+/* Cor azul para o link "Laptops & Computers" */
+.default-blue {
+  color: #00acdc;
```

Comment:
The color #00acdc is used for the 'default-blue' class. Ensure that this color provides sufficient contrast against the background for accessibility, especially for users with visual impairments.